examples/custom-buildx/README.md

```diff
@@ -0,0 +1,126 @@
+```
+
+##### Step 2: Configure the custom builder
+
+Take a look at the [`skaffold.yaml`](skaffold.yaml), which uses a
+_custom builder_
+```yaml
+  - image: skaffold-examples-buildx
+    custom:
+      buildCommand: sh buildx.sh
+      dependencies:
+        paths: ["go.mod", "**.go", "buildx.sh"]
+```
+
+Simple build commands can be inlined into the `skaffold.yaml`, but
+in this example we have created a separate [`build.sh`](build.sh)
+script.  This script uses `docker buildx` to containerize
+source code for two platforms, `linux/amd64` and `linux/arm64`.
+
+For more information on configuring a custom builder, see the Skaffold custom
+builder [documentation](https://skaffold.dev/docs/how-tos/builders/#custom-build-script-run-locally).
+Note that Skaffold builders are different from `docker buildx` builders.
+
+
+##### Step 3: Configure node affinities
+
+Next look at the Kubernetes [pod descriptor](k8s/pod.yaml) and notice
+the use of _node affinities_ to instruct Kubernetes to schedule the workload
+on nodes running either `linux/arm64` or `linux/amd64`.  It is important
+to realize that Kubernetes does not examine the referenced container images
+to determine the possible platforms.
+
+
+##### Step 4: Build and deploy the example
+
+Now, use Skaffold to deploy this application to your Kubernetes cluster:
+
+```shell
+$ skaffold run --tail --default-repo <your repo>
+```
+
+With this command, Skaffold will build the artifact with `docker buildx`
+and deploy the application to Kubernetes.  You should be able to
+see *Hello, World from <OS><ARCH>!* printed every second in the Skaffold logs.
+
+We need to use `--default-repo` to push to a repository as the
+Docker Daemon does not support loading multi-platform images with
+the same name.
+
+
+#### Cleanup
+
+To clean up your Kubernetes cluster, run:
+
+```shell
+$ skaffold delete
+```
+
+
+##### &#x26A0; Caution &#x26A0;
+
+Using `buildx` to build for multiple platforms has some subtle
+interactions with Skaffold's artifact caching.
+
+Skaffold normally caches the artifact after a successful container
+image build.  Using the principle that given the same source inputs,
+a build should produce the same container image, Skaffold records
+an association of the resulting image digest with a hash of the
+artifact source.  This is the same principle used followed Docker
+and other builders to cache image layers to speed up builds.
+
+In this example, the build script configures Buildx differently
+depending on the `$PUSH_IMAGE` flag. 
+
+  - When `true`, the result is to be pushed to a registry, and all
+    registries support multi-platform images. 
+  - When `false`, the result is to be loaded to the Docker Daemon. 
+    But Buildx does not support loading multi-platform images to
+    the Docker Daemon, and so this example builds a single container
+    image for the local platform.
+
+But Skaffold is unaware that the build result differs based on `$PUSH_IMAGE`.
```

Comment:
```suggestion 
Skaffold is unaware that the build result differs based on `$PUSH_IMAGE`.
```

Reply:
The "But" is helpful here as it is contrary to expectations.

Reply:
they were too many buts. There was a but in the sentence above too. I am not an expert on english **but* in my mind  all sentences after the first "But" inherit the "but" clause?